pyphare/pyphare/pharesee/plotting.py

```
@@ -53,50 +53,111 @@ def dist_plot(particles, **kwargs):
-    if "gaussian_filter_sigma" in kwargs and "median_filter_size" not in kwargs:
-        from scipy.ndimage import gaussian_filter
+        drawstyle = kwargs.get("drawstyle", "steps-mid")
 
-        sig = kwargs.get("gaussian_filter_sigma", (0, 0))
-        image = gaussian_filter(h.T, sigma=sig)
-    elif "median_filter_size" in kwargs and "gaussian_filter_sigma" not in kwargs:
-        from scipy.ndimage import median_filter
+        if axis[0] in vaxis:
+            x = new_particles.v[:, vaxis[axis[0]]]
+        else:
+            raise ValueError("For 1-D dist_plot, the abscissa has to be a velocity axis")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)